src/NodaTime/LocalDate.cs

```diff
@@ -329,6 +334,7 @@ public static LocalDate FromYearMonthWeekAndDay(int year, int month, int occurre
 
         /// <summary>
         /// Subtracts the specified period from the date.
+        /// Fields are subtracted in descending order of significance (years first, then months etc).
```

Comment:
Incidentally, this surprises me. I guess I would have expected that `date + period - period == date`, but that presumably isn't true, given this?

Reply:
That isn't always true even for a single field. Consider January 30th + 1 month - 1 month - that will always result in January 28th or January 29th. Sigh.